site/overview/faqs.md

```diff
@@ -0,0 +1,158 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Frequently Asked Questions
+  - - meta
+    - name: description
+      content: Learn how to get started with Alchemy's Account Kit, a vertically integrated stack for building apps that support ERC-4337.
+  - - meta
+    - property: og:description
+      content: Frequently Asked Questions
+  - - meta
+    - name: twitter:title
+      content: FAQs
+  - - meta
+    - name: twitter:description
+      content: Frequently Asked Questions
+---
+
+# Frequently Asked Questions
+
+## Smart Accounts - Light Account
+
+### Do accounts have the same address across all chains?
+
+::: details Answer
+In almost all cases, yes, you will get the same address on all chains as long as the owner address is the same! The deployment address is a function of the owner address, the account implementation (e.g. latest version of Light Account), and the salt (you can optionally specify this). If all three of those remain the same, then you deploy the smart account at the same contract address.
+
+There are two scenarios where you'd get a different contract address:
+
+1. If you deploy one smart account, then change the owner address, then deploy the second account.
+2. If you upgrade the smart account (e.g. to a new version of Light Account). It is unlikely that we will make many updates to this contract so the address will not change frequently.
+   :::
+
+### How would Alchemy initiate an upgrade of a Light Account?
+
+::: details Answer
+It is unlikely we will frequently update the Light Account contract itself, however it is possible if needed. Light Account has [`UUPSUpgradeable`](https://github.com/alchemyplatform/light-account/blob/main/src/LightAccount.sol#L50) which adds upgrade methods on the account itself. To upgrade an account you will need to send a `UserOperation` using that method.
+:::
+
+### Can I have multiple accounts for the same owner address? / How do I set the value of the salt/index for Light Account?
+
+::: details Answer
+Yes! The optional index value (salt) on Light Account enables the ability to have multiple accounts for the same owner address. This value defaults to 0. You can set it in the [constructor](/packages/aa-accounts/light-account/constructor.html#params-simplesmartaccountparams).
+:::
+
+## Submitting User Operations
+
+### How does the speed of `UserOperation`s compare to normal transactions?
+
+::: details Answer
+If the `UserOperation` (meta-transaction for 4337 accounts) is correctly priced and submitted a few hundred milliseconds (ms) prior to a new block getting created, it will typically get placed in the next block. This is because the bundler needs time to create/propagate its transaction. You can think of it as 1 extra block time worth of latency, but we are working towards improving this latency.
```

Comment:
```suggestion
If the `UserOperation` (meta-transaction for 4337 accounts) is correctly priced and submitted a few hundred milliseconds (ms) prior to a new block getting created, it will typically get placed in the next block. This is because the Bundler needs time to create/propagate its transaction. You can think of it as 1 extra block time worth of latency, but we are working towards improving this latency.
```